src/main/java/com/github/chainmailstudios/astromine/registry/AstromineItems.java

```diff
@@ -60,6 +61,7 @@
 
 	// Fantasy weaponry
 	public static final Item SUPER_SPACE_SLIME_SHOOTER = register("super_space_slime_shooter", new SuperSpaceSlimeShooterItem(getBasicSettings()));
+	public static final GravityGauntletItem GRAVITY_GAUNTLET = register("gravity_gauntlet", new GravityGauntletItem(getBasicSettings().fireproof().maxCount(1), AstromineConfig.get().gravityGauntletEnergy));
```

Comment:
I would move the damage() to the constructor of GravityGauntletItem like vanilla and what's the point of fireproof

Reply:
to match the other weapons

Reply:
also the gauntlet doesnt take damage

Reply:
don't think we want fire proof on a guantlet

Reply:
ok